backend/src/core/schemas/user/base.py

```diff
@@ -39,6 +39,22 @@ class UserCreate(UserBase):
     instagram_link: str | None = None
     free_dates: list[date] | None = None
 
+    @field_validator("free_dates")
+    def validate_free_dates(cls, value):
+        if value:
+            for free_date in value:
+                if free_date < date.today():
+                    raise ValueError(""" Field "free_dates" cannot contain previos days """)
+
+        return value
```

Comment:
Let's keep things annotated please + it would be great to specify validator mode event if it's default one
```suggestion
    @field_validator("free_dates", mode="after")
    def validate_free_dates(cls, value: list[date] | None) -> list[date] | None:
        if value:
            for free_date in value:
                if free_date < date.today():
                    raise ValueError(""" Field "free_dates" cannot contain previos days """)

        return value
```

Here we want to apply our validation after pydantic makes types checking so we use mode 'after'